### PR TITLE
Move logo on header & remove footer

### PR DESCRIPTION
**Pull request by thundron**
Change-type: patch
Changelog-entry: Move logo on header & remove footer
Signed-off-by: Lorenzo Alberto Maria Ambrosi <lorenzothunder.ambrosi@gmail.com>

### CODE REVIEW

File: lib/gui/app/components/balena-etcher-logo/balena-etcher-logo.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+// eslint-disable-next-line no-unused-vars
+const React = require('react')
+const { open } = require('../../os/open-external/services/open-external')
+const SvgIcon = require('../svg-icon/svg-icon.jsx')
+const {
+  default: styled
+} = require('styled-components')
+
+const Div = styled.div `
```
Comment (zvin) [inline lib/gui/app/components/balena-etcher-logo/balena-etcher-logo.jsx]: Can we use a more explicit name? `CenteredDiv` maybe?

File: lib/gui/app/index.html [hunk 2]
```diff
@@ -11,6 +11,17 @@
   </head>
   <body>
     <header class="section-header" ng-controller="HeaderController as header">
+
+      <balena-etcher-logo>
+      </balena-etcher-logo>
+
+      <!-- <div os-open-external="https://www.balena.io?ref=etcher"
```
Comment (zvin) [inline lib/gui/app/index.html]: Commented out html.

File: lib/gui/app/index.html [hunk 3]
```diff
@@ -35,41 +46,16 @@
...
-        os-open-external="https://github.com/balena-io/etcher/blob/master/CHANGELOG.md"></span>
-    </footer>
-
     <div class="section-loader"
       ng-controller="StateController as state"
       ng-class="{
         isFinish: state.currentName === 'success'
       }">
-      <safe-webview
+      <!-- <safe-webview
```
Comment (zvin) [inline lib/gui/app/index.html]: Leftover commented out html

File: lib/gui/app/index.html [hunk 4]
```diff
@@ -35,41 +46,16 @@
...
         src="'https://www.balena.io/etcher/success-banner/'"
+        refresh-now="state.previousName === 'success'"></safe-webview> -->
+      <safe-webview
+        src="'http://localhost:8000/etcher/success-banner'"
```
Comment (zvin) [inline lib/gui/app/index.html]: localhost url

File: lib/gui/app/pages/finish/templates/success.tpl.html [hunk 5]
```diff
@@ -21,24 +21,6 @@
...
-          by
-          <span os-open-external="https://resin.io?ref=etcher_success">
-            <svg-icon paths="[ '../../assets/balena.svg' ]"
-              width="'auto'"
-              height="'20px'">
-            </svg-icon>
-          </span>
-          <div class="section-footer">
-            <span class="caption caption-small footer-right"
-              manifest-bind="version"
```
Comment (lurch) [inline lib/gui/app/pages/finish/templates/success.tpl.html]: Does this change mean that the version number won't be displayed anywhere in the application?

Comment (thundron) [reply lib/gui/app/pages/finish/templates/success.tpl.html]: No, it's gonna be included in the settings, still working on it really, should've made this a `draft` PR

**Pull request closed without merging**